SharePointFramework/PortfolioWebParts/src/components/PortfolioOverview/reducer.ts

```diff
@@ -229,9 +229,18 @@ const $createReducer = (params: IPortfolioOverviewReducerParams) =>
           return isSortedDescending ? $a - $b : $b - $a
         })
       } else {
-        state.items = sortArray(state.items, [payload.column.fieldName], {
-          reverse: !isSortedDescending
-        })
+        if (payload.column.dataType === 'currency') {
```

Comment:
Foretrekker switch-case her.

```
switch(payload.column.dataType) {
     ....
}
```

Reply:
Er ikke tid at the moment til å få gjennomført dette nå. Tenker for læringsdel så gjør vi dette til en ny PR som kan jobbes med av Håvard i juli.

Reply:
Det tok meg nøyaktig 1m 13s.